### DISCUSSION TITLE
Offer CloudApp export extension

### DISCUSSION THREAD

**Discussion by maxengel** [standardnotes/forum#2506] [Suggestions] [answered]
[Cloup App](https://www.getcloudapp.com/) renders Markdown incredibly well and makes the notes easy to share. I've been downloading notes, changing the extension from .txt to .md, and then uploading it. Instead, it would be great if an option when a Markdown editor is enabled if there was an option to export it to CloudApp.

Their API is [here](https://get.cloudapp.help/hc/en-us/articles/223483088-Does-CloudApp-have-an-API-).

**Comment by atmoio** [comment]
Interesting. That would be useful. I don't imagine this is something I would build myself, but it would be easy for any contributor who wants to build this to get started.

**Comment by maxengel** [comment]
How could I start a exploring building this?

**Comment by atmoio** [comment]
Check out the Listed extension, which does sharing. I don't have a tutorial up yet, so for now you'll have to study the source: https://github.com/standardnotes/listed/blob/master/app/controllers/authors_controller.rb#L71

I hope to get a tutorial out soon.

**Comment by myreli** [comment, marked as answer]
While directly sharing with CloudApp isn't supported, SN now offers the ability to export multiple notes to Markdown (when using the Super Editor)

<img width="1350" alt="image" src="https://user-images.githubusercontent.com/17554234/218585804-916bc8fa-0a15-461b-a387-943029a2f2b4.png">


I'm triaging older issues so I will mark it as resolved, but feel free to open it again if necessary.
